Runtime/JSArray.cs

```diff
@@ -0,0 +1,41 @@
+using System.Collections;
+using System.Collections.Generic;
+
+namespace NodeApi;
+
+public struct JSArray : IEnumerable<JSValue>, IEnumerable
```

Comment:
`IEnumerable<T>` extends `IEnumerable` so it's not necessary to declare the latter. Same for the other JS collections.

Reply:
It is removed.